src/pages/Main/MainSlick.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import Slider from 'react-slick';
+import styled from 'styled-components/macro';
+
+function NextArrow(props) {
+  const { className, style, onClick } = props;
+  return (
+    <img
+      src="images/next.png"
+      alt="next.png"
+      className={className}
+      style={{
+        ...style,
+        width: '50px',
+        height: '50px',
+        display: 'block',
+        right: '-75px',
+      }}
+      onClick={onClick}
+    />
+  );
+}
+
+function PrevArrow(props) {
+  const { className, style, onClick } = props;
+  return (
+    <img
+      src="images/prev.png"
+      alt="next.png"
+      className={className}
+      style={{
+        ...style,
+        display: 'block',
+        width: '50px',
+        height: '50px',
+        left: '-75px',
+      }}
+      onClick={onClick}
+    />
+  );
+}
+
+const MainSlick = () => {
+  const [restaurant, setRestaurant] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/mainSlickData.json')
+      .then(res => res.json())
+      .then(setRestaurant);
+  });
```

Comment:
```suggestion
  useEffect(() => {
    fetch('/data/mainSlickData.json')
      .then(res => res.json())
      .then(setRestaurant);
      console.log(1)
  });
```
이렇게 해보시고, console탭에서 어떤 일이 일어나고있는지 확인해 보시고, 수정 해 주세요